### PR TITLE
A Pull Request on JavaScript Capstone Project.

### PR DESCRIPTION
**Pull request by atok624**
## In this Pull Request, we did the following:
- Created the basic shared code on the home page.
- Started individual interfaces.
- Gave code reviews to your teammates.
- Completed the work on individual interfaces.
- Recorded a video for your project.
- Created a good README and PR description.
- Asked for the external review - using your Student Dashboard.

### CODE REVIEW

File: src/comments.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                      <p class="comment"><span class="origin">Country</span>: ${detail.network.country.name}</p>
+                      <p class="comment"><span class="origin">Language</span>: ${detail.language}</p>
+                  </div>
+              </div>
+              <div class="comment-wrap">
+                  <p>comments <span id="comments-count">${(JSON.parse(comments) && JSON.parse(comments).length) ? JSON.parse(
+              comments,
+            ).length : 0}</span></p>
+              </div>
+              <ul class="commentors">${detailResult}</ul>
```
Comment (binhussen) [inline src/comments.js:105]: - Kindly, we need need to refresh the page to view the newly added comment, so please update it to have better user interaction :+1: 
Screenshot from 2023-03-17 23-01-29

File: test.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  test('update items count', () => {
+    updateCount(itemsArray);
+    expect(count).toBe(20);
+  });
+
+  test('update comments count', () => {
+    addComment();
+    expect(commentsCount).toBe(1);
+  });
+});
```
Comment (binhussen) [inline test.test.js:50]: - Kindly, you need to test here the counter function, that has been created for counting the number of movies and comments :+1:

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @atok624  and @ernestmusong , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Used web pack and jest ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by atok624** [comment]
Thank you @binhussen 
We will surely implement all the changes

**Review by topeogunleye — APPROVED** [review]
### Hi @ernestmusong, @atok624, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by atok624** [comment]
Thank you so much @topeogunleye.
 We were patiently waiting for some good news and this is the best news of the day for us.
Thank you.

**Comment by topeogunleye** [comment]
> Thank you so much @topeogunleye. We were patiently waiting for some good news and this is the best news of the day for us. Thank you.

@ernestmusong you're welcome

**Pull request merged**